includes/validation/class-amp-validation-manager.php

```diff
@@ -245,6 +246,20 @@ function( $query_vars ) {
 			add_action( 'all_admin_notices', [ __CLASS__, 'print_plugin_notice' ] );
 
 			add_action( 'admin_bar_menu', [ __CLASS__, 'add_admin_bar_menu_items' ], 101 );
+
+			add_action(
+				'admin_bar_init',
+				static function () {
+					wp_enqueue_style(
+						'amp-icons',
+						amp_get_asset_url( 'css/amp-icons.css' ),
+						[ 'dashicons', 'admin-bar' ],
+						AMP__VERSION
+					);
+
+					wp_styles()->add_data( 'amp-icons', 'rtl', 'replace' );
+				}
+			);
```

Comment:
@pierlon I found an issue. This is not currently getting run in Reader mode. When Reader mode is selected, then no Link icon appears in the header.

I tried moving this code to `amp_init` and that fixed the issue, but then it revealed two others:

1. The `amp-icons` stylesheet here is not only used for the admin bar but also for other parts of the screens.
2. The stylesheet was being added to every screen in the admin (and the frontend) regardless of whether it was being used.

I've implemented a fix in c4cb9ab47f639070a8410c2fa4cf64da1c20c377.